packages/features/bookings/Booker/components/EventMeta.tsx

```diff
@@ -101,6 +106,31 @@ export const EventMeta = () => {
                 </span>
               )}
             </EventMetaBlock>
+            {bookerState === "booking" && seatedEventData.seatsPerTimeSlot ? (
+              <EventMetaBlock
+                icon={User}
+                className={`${
+                  seatedEventData?.attendees || 0 / seatedEventData.seatsPerTimeSlot >= 0.5
+                    ? "text-rose-600"
+                    : seatedEventData?.attendees || 0 / seatedEventData.seatsPerTimeSlot >= 0.33
+                    ? "text-yellow-500"
+                    : "text-bookinghighlight"
+                }`}>
+                <div className="text-bookinghighlight flex items-start text-sm">
```

Comment:
I don't think these text- colors exist anymore? Do they get added by embed @hariombalhara ?

Reply:
It looks like they still work? What should we update to? @sean-brydon

Reply:
<img width="600" alt="Screenshot 2023-07-01 at 3 51 12 PM" src="https://github.com/calcom/cal.com/assets/1780212/b5a188b8-593d-4c4e-a95d-0db055bd7095">
They are used in Old Booker only. So, I would say just remove this class and see if anything breaks and if not then we are okay removing it.